library/snapshot.py

```diff
@@ -1,22 +1,146 @@
-from __future__ import print_function
+#!/usr/bin/python
```

Comment:
Do you plan to add python unit tests for this module?

Reply:
Yes.  I'll add some unit tests.